src/Skia/Avalonia.Skia/DrawingContextImpl.cs

```diff
@@ -915,90 +915,117 @@ private static void ConfigureGradientBrush(ref PaintWrapper paintWrapper, Rect t
                     break;
                 }
                 case IRadialGradientBrush radialGradient:
-                {
-                    var centerPoint = radialGradient.Center.ToPixels(targetRect);
-                    var center = centerPoint.ToSKPoint();
-                    
-                    var radiusX = (radialGradient.RadiusX.ToValue(targetRect.Width));
-                    var radiusY = (radialGradient.RadiusY.ToValue(targetRect.Height));
-
-                    var originPoint = radialGradient.GradientOrigin.ToPixels(targetRect);
-                    
-                    Matrix? transform = null;
-                    
-                    if (radiusX != radiusY)
-                        transform =
-                            Matrix.CreateTranslation(-centerPoint)
-                            * Matrix.CreateScale(1, radiusY / radiusX)
-                            * Matrix.CreateTranslation(centerPoint);
-                    
-                    
-                    if (radialGradient.Transform != null)
                     {
```

Comment:
Formatting nit: the whole case has an extra indentation level.

Reply:
This is done by VS for some reason. Ideally I would want to split the method into multiple parts to better organize it.